runtime/lua/vim/treesitter/dev.lua

```diff
@@ -42,11 +42,32 @@ local TSTreeView = {}
 ---@param lang string Language of the tree currently being traversed
 ---@param injections table<string, TSP.Injection> Mapping of node ids to root nodes
 ---                  of injected language trees (see explanation above)
+---@param include_root boolean Whether to include the root nodes in the tree
```

Comment:
Let's not add extra knobs and configurations here. Just make it the default and not configurable.

One feature we could add in the future is the ability to "reparent" the tree at any node. So with the tree view open you press a key on any node, and a new tree view is opened that shows only nodes that are children of the node that the key was pressed on. If we had such a feature, then users could use that on the top-level root node to hide it, if they really don't want to see it.

Reply:
+1